corehq/apps/sso/forms.py

```diff
@@ -583,9 +591,40 @@ class BaseSsoEnterpriseSettingsForm(forms.Form):
         label=gettext_lazy("Entity ID"),
         required=False,
     )
+    enable_user_deactivation = forms.BooleanField(
+        required=False,
+        label=gettext_lazy("Auto-Deactivation"),
+        help_text=gettext_lazy(
+            "This option ensures any authorization provided by CommCare "
+            "HQ to Web Users using SSO is removed when those users are removed "
+            "from Azure AD (Entra ID). This includes automatic deactivation of any "
+            "API keys associated with these users."
+        ),
+        widget=BootstrapCheckboxInput(
+            inline_label=gettext_lazy("Automatically deactivate Web Users")
+        )
+    )
+    api_host = forms.CharField(required=False, label=gettext_lazy("Tenant Id"))
+    api_id = forms.CharField(required=False, label=gettext_lazy("Application ID"))
+    api_secret = forms.CharField(required=False, label=gettext_lazy("Client Secret"))
+    date_api_secret_expiration = forms.DateTimeField(
+        required=False,
+        label=gettext_lazy("Secret Expires On")
+    )
 
     def __init__(self, identity_provider, *args, **kwargs):
+        if 'show_remote_user_management' in kwargs:
+            self.show_remote_user_management = kwargs.pop('show_remote_user_management')
+        else:
+            self.show_remote_user_management = False
```

Comment:
`show_remote_user_management` should be set in `SsoSamlEnterpriseSettingsForm` since it's for the feature flag and only needed by that form.

Reply:
You're right. Thank you! Fixed in 44ec50ec32f8c39711f6bc8ad5271b2a5da3f088